cfg/osic1p.cfg

```diff
@@ -2,24 +2,28 @@ FEATURES {
     STARTADDRESS: default = $0200;
 }
 SYMBOLS {
+# Add "-u __BOOT__" to a command line if you want a file that can be loaded directly.
+#    __BOOT__:      type = import;
```

Comment:
The upper-case `-U` option undefines names.  The lower-case `-u` option "uses" names (i.e., it makes the linker import them).

I included those commented lines as a convenience to programmers.  If they never use **C1Pjs**, then they _simply_ could uncomment the lines in the two configure files; they wouldn't need to type a full line into those files.
I added a mention about it in the documentation.

P.S.  I also like the symmetry of having it there; it's a companion of the `HEAD:` and `BOOT:` lines.